gpm_api/tests/test_io/test_download.py

```diff
@@ -306,11 +309,11 @@ def test_check_download_status(
 
 
 def test_get_fpaths_from_fnames(
+    mock_configuration: Dict[str, str],
```

Comment:
There is a `TODO: WRONG REDO` here below. Still actual?

Reply:
The test passes, but I still need to go through testing the whole `download.py` (plus `pps` and `ges_disc`) to be sure that the test covers what it should